3.10/aql/functions-geo.md

```diff
@@ -45,14 +45,40 @@ FOR doc IN doc // e.g. documents returned by a traversal
 `GEO_CONTAINS(geoJsonA, geoJsonB) → bool`
 
 Checks whether the [GeoJSON object](../indexing-geo.html#geojson) `geoJsonA`
-fully contains `geoJsonB` (Every point in B is also in A). The object `geoJsonA` has to be of type 
-`Polygon` or `MultiPolygon`, other types are not supported because containment is ill defined. 
-This function can be **optimized** by a S2 based [geospatial index](../indexing-geo.html).
+fully contains `geoJsonB` (every point in B is also in A). The object `geoJsonA`
+has to be of type `Polygon` or `MultiPolygon`, other types are not supported
+because containment is ill-defined.
 
 - **geoJsonA** (object): first GeoJSON object or coordinate array (in longitude, latitude order)
 - **geoJsonB** (object): second GeoJSON object or coordinate array (in longitude, latitude order)
-- returns **bool** (bool): true when every point in B is also contained in A, false otherwise
+- returns **bool** (bool): true if every point in B is also contained in A, false otherwise
 
+{% hint 'info' %}
+ArangoDB faithfully shows the same behavior as the underlying
+S2 geometry library in the following sense. The S2 documentation states:
+
+> Point containment is defined such that if the sphere is subdivided
+> into faces (loops), every point is contained by exactly one face.
+> This implies that loops do not necessarily contain their vertices.
+
+As a consequence, a loop or polygon does not necessarily contain its
+boundary edges.
+{% endhint %}
+
+A query containing a `FILTER` expression of the form
+
+```js
+FOR doc IN collectionname
+  FILTER GEO_CONTAINS(geoJson, doc.geo)
+  ...
+```
+
+can be **optimized** by an S2-based [geospatial index](../indexing-geo.html)
+(on the attribute `geo` of the collection `collectionname` in this example), if
```

Comment:
The conditional 'if' at the end reads difficult. Perhaps 
"...in this example), **only** if ...  "
After the conditions of IF, I'd add a line break.